src/components/base/JAssetSymbol/JAssetSymbol.js

```diff
@@ -11,26 +11,35 @@ type JAssetSymbolColor = 'blue' | 'gray' | 'white'
 type Props = {
   symbol: string,
   color: JAssetSymbolColor,
+  isCustom?: boolean,
 }
 
-const getSymbolClassName = symbol =>
-  SYMBOLS_AVAILABLE_CLASS_INDEX[symbol.toLowerCase()] || '-symbol-not-listed'
+const getSymbolClassName = (symbol, isCustom) =>
+  !isCustom && SYMBOLS_AVAILABLE_CLASS_INDEX[symbol.toLowerCase()] ? '' : '-symbol-not-listed'
 
 function JAssetSymbol({
-  symbol, color,
+  symbol, color, isCustom,
 }: Props) {
   const url = iconsAsset[`${symbol.toLowerCase()}-usage`]
 
   return (
     <div
-      className={classNames(`j-asset-symbol ${getSymbolClassName(symbol)} -${color}`)}
-      data-symbol={symbol}
+      className={classNames(`j-asset-symbol ${getSymbolClassName(symbol, isCustom)} -${color}`)}
+      data-symbol={symbol[0]}
```

Comment:
Final business resolution:
- if there is less than 5 chars, display all of them
- if there are 5 chars or more, display first 3 only

Reply:
🔢